repos/system_upgrade/el8toel9/actors/checkvdo/libraries/checkvdo.py

```diff
@@ -135,7 +141,11 @@ def _process_undetermined_conversion_devices(devices):
 
 
 def check_vdo(conversion_info):
-    all_vdo_converted = api.current_actor().get_all_vdo_converted_response()
+    if not _lvm_package_installed():
+        # nothing to do. VDO does not work without LVM
+        return
+
+    all_vdo_converted = api.current_actor().get_vdo_answer()
```

Comment:
For completeness...
`get_vdo_answer` isn't part of this push, but I know it's coming. :-)

Reply:
aa. yep :)